examples/css/example.scss

```diff
@@ -37,7 +35,13 @@ body{
     }
     margin: 0 0 40px 0;
     text-align: center;
-
+  }
+  h2 {
+    font-size: 1.2em;
+    margin: 0 0 0.5em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 0.5em` instead of `0 0 0.5em 0`